configure

```diff
@@ -278,15 +278,15 @@ if [ -n "$NETCDF4" ] ; then
     if [ -f $NETCDF/bin/nf-config ] ; then
       nx_config="$NETCDF/bin/nf-config --flibs"
       DEP_LIB_PATH="`$nx_config | awk '{for(i=1;i<=NF;i++){if(match($i, /-L.*/)) {print $i} } }'`"
-      CURL="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i == "-lcurl") {print $i} } }'`"
-      GPFS="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i == "-lgpfs") {print $i} } }'`"
+      CURL="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i = "-lcurl") {print $i} } }'`"
+      GPFS="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i = "-lgpfs") {print $i} } }'`"
     fi
     if [ "$DEP_LIB_PATH" = '' ] ; then
       if [ -f $NETCDF/bin/nc-config ] ; then
         nx_config="$NETCDF/bin/nc-config --libs"
         DEP_LIB_PATH="`$nx_config | awk '{for(i=1;i<=NF;i++){if(match($i, /-L.*/)) {print $i} } }'`"
-        CURL="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i == "-lcurl") {print $i} } }'`"
-        GPFS="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i == "-lgpfs") {print $i} } }'`"
+        CURL="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i = "-lcurl") {print $i} } }'`"
+        GPFS="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i = "-lgpfs") {print $i} } }'`"
```

Comment:
As above, the modified code here is an `awk` script and not shell, so we may want to consider reverting these changes.